engine/src/main/java/org/camunda/bpm/engine/impl/jobexecutor/JobExecutor.java

```diff
@@ -96,7 +96,7 @@ public synchronized void shutdown() {
   }
 
   protected void ensureInitialization() {
-    acquireJobsCmdFactory = new DefaultAcquireJobsCommandFactory(this);
+    acquireJobsCmdFactory = acquireJobsCmdFactory == null ? new DefaultAcquireJobsCommandFactory(this) : acquireJobsCmdFactory;
```

Comment:
This is a suggestion to make the code more readable:

```suggestion
  if (acquireJobsCmdFactory == null) {
    acquireJobsCmdFactory =  new DefaultAcquireJobsCommandFactory(this);
  }
```